addons/bus/static/tests/mock_server/mock_models/ir_websocket.js

```diff
@@ -14,7 +15,7 @@ export class IrWebSocket extends models.ServerModel {
     _updatePresence(inactivityPeriod, imStatusIdsByModel) {
         const imStatusNotifications = this._getImStatus(imStatusIdsByModel);
         if (Object.keys(imStatusNotifications).length > 0) {
-            const [partner] = this.env["res.partner"].read(this.env.partner_id);
+            const [partner] = this.env["res.partner"].read(session.partner_id);
```

Comment:
Reading something inside the "server" from the JS session creates a coupling that feels very off.